website/project/metadata/test-schema.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Since we had to move away from using the `prereg-prize.json` as a test value I put in a generic test schema